pkgs/applications/science/math/eukleides/default.nix

```diff
@@ -34,19 +38,16 @@ lib.fix (eukleides: stdenv.mkDerivation rec {
 
   postInstall = ''
     wrapProgram $out/bin/euktoeps \
-      --set-default TEXINPUTS : \
-      --prefix TEXINPUTS : "$tex/tex/latex/eukleides" \
-      --prefix PATH : "${texLive}/bin"
-    wrapProgram $out/bin/euktopdf \
-      --set-default TEXINPUTS : \
-      --prefix TEXINPUTS : "$tex/tex/latex/eukleides" \
-      --prefix PATH : "${texLive}/bin"
+      --prefix PATH : ${getopt}/bin
   '';
 
   outputs = [ "out" "doc" "tex" ];
 
   passthru.tlType = "run";
-  passthru.pkgs = [ eukleides.tex ];
+  passthru.pkgs = [ eukleides.tex ] ++
+    # packages needed by euktoeps, euktopdf and eukleides.sty
+    (with texlive; collection-pstricks.pkgs ++ epstopdf.pkgs
+      ++ iftex.pkgs ++ moreverb.pkgs);
```

Comment:
```suggestion
  passthru.pkgs = [ eukleides.tex ]
    # packages needed by euktoeps, euktopdf and eukleides.sty
    ++ (with texlive; collection-pstricks.pkgs ++ epstopdf.pkgs ++ iftex.pkgs ++ moreverb.pkgs);
```

Reply:
Thanks! Is this coming from `nixpkgs-fmt` or something similar? Happy to use it in the future, if that's the advice.

Reply:
No, I just shortened it a bit.